packages/playback-core/src/index.ts

```diff
@@ -558,6 +561,152 @@ export const getStreamTypeConfig = (streamType?: ValueOf<StreamTypes>) => {
   return {};
 };
 
+export const getDRMConfig = (
+  props: Partial<Pick<MuxMediaPropsInternal, 'src' | 'playbackId' | 'drmToken' | 'customDomain'>>
+) => {
+  const {
+    drmToken,
+    src,
+    playbackId = toPlaybackIdFromSrc(src), // Since Mux Player typically sets `src` instead of `playbackId`, fall back to it here (CJP)
+  } = props;
+  if (!drmToken || !playbackId) return {};
+  return {
+    emeEnabled: true,
+    // licenseXhrSetup: (
```

Comment:
This will likely be unneeded but leaving here for reference until we've completed impl full stack, just in case.